app/assets/src/components/views/phylo_tree/PhyloTreeByPathCreation.jsx

```diff
@@ -0,0 +1,405 @@
+import PropTypes from "prop-types";
+import React from "react";
+import Input from "../../ui/controls/Input";
+import Wizard from "../../ui/containers/Wizard";
+import axios from "axios";
+import DataTable from "../../visualizations/table/DataTable";
+import Moment from "react-moment";
+
+const MinNumberOfSamples = 4;
+class PhyloTreeByPathCreation extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      defaultPage: 0,
+      skipListTrees: false,
+      phyloTreesLoaded: false,
+      phyloTrees: [],
+
+      samplesLoaded: false,
+      projectSamples: [],
+      selectedProjectSamples: new Set(),
+
+      otherSamples: [],
+      selectedOtherSamples: new Set(),
+      otherSamplesFilter: "",
+
+      showErrors: false,
+      treeName: ""
+    };
+
+    this.phyloTreeHeaders = {
+      name: "Phylogenetic Tree",
+      user: "Creator",
+      last_update: "Last Updated",
+      view: "View"
+    };
+
+    this.projectSamplesHeaders = {
+      name: "Name",
+      host: "Host",
```

Comment:
For now, maybe comment Host/Tissue/Location out so that it's not shown in the modal. I'll fill it in with the actual data later